src/main/java/org/sciborgs1155/robot/vision/VisionConstants.java

```diff
@@ -0,0 +1,28 @@
+package org.sciborgs1155.robot.vision;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N3;
+
+public class VisionConstants {
+  public static final AprilTagFieldLayout TAG_LAYOUT = null;
+
+  public static final String FRONT_CAMERA_NAME = "placeholder";
```

Comment:
again, we need a `CameraConfig` record, with two instances, one for either camera, here